Formula/t/teller.rb

```diff
@@ -1,8 +1,8 @@
 class Teller < Formula
   desc "Secrets management tool for developers built in Go"
```

Comment:
> `built in Go`

That doesn't seem right

Reply:
Description update PR here: https://github.com/Homebrew/homebrew-core/pull/172132